src/components/countries.js

```diff
@@ -17,38 +19,43 @@ const Countries = () => {
 
   return (
     <div>
-      <h1>All Countries</h1>
-      <div>
+      {' '}
+      <div className="dropdown">
+        <img src={worldwide} alt="Globe" />
+        <h1 className="title">World</h1>
```

Comment:
[OPTIONAL] I think, it would be better if you add some metrics similar to the views on the hero section. Also, it would be better if you align the hero section to the center for the desktop version. 👍  In this way, you will have a consistent and responsive web app 👍